reflex/reflex.py

```diff
@@ -78,6 +78,8 @@ def _init(
     app_name = prerequisites.get_default_app_name() if name is None else name
     console.rule(f"[bold]Initializing {app_name}")
 
+    prerequisites.check_latest_package_version("reflex")
```

Comment:
Instead of "reflex" can use `constants.Reflex.MODULE_NAME` (below also). Maybe we can make a new constant for the "reflex-hosting-cli" within there also